### DISCUSSION TITLE
Database broadlink

### DISCUSSION THREAD

**Discussion by Vienna15** [indomus/forum#2317] [HUB: Home Assistant]
Buongiorno vi scrivo per chiedervi come faccio a trovare i database per i telecomandi del mio condizionatore hitachi (rar-7pe1) e Sanier Douval (yam1f). Ho un Rm3 pro broadlink e mini. Se non ci sono i database come posso estrapolare?

**Comment by KC-inDomus** [comment]
> Buongiorno vi scrivo per chiedervi come faccio a trovare i database per i telecomandi del mio condizionatore hitachi (rar-7pe1) e Sanier Douval (yam1f). Ho un Rm3 pro broadlink e mini. Se non ci sono i database come posso estrapolare?

scusa, ma hai il telecomando originale?

**Comment by Vienna15** [comment]
si tutti e due

**Reply by KC-inDomus** [reply]
allora acquisici i codici e fine https://indomus.it/progetti/domotizzare-un-condizionatore-tradizionale-con-broadlink-e-home-assistant-v2/

**Reply by Vienna15** [reply]
Scusa l'ignoranza una volta che acquisisco i codici come l'implemento in app broadlink?

**Reply by KC-inDomus** [reply]
l'archivio collaborativo è per utenti homebridge e home assistant, non per chi usa l'app broadlink. avevo dato per scontato fossi tra quelli

**Comment by Vienna15** [comment]
ma non ci sono in archivio e se faccio matching e' tutto sballato

**Reply by Vienna15** [reply]
In più si può fare con IP dinamico? Non vorrei modificarlo in statico

**Comment by Vienna15** [comment]
Scusami. Sono scarso in materia

Il dom 5 feb 2023, 09:12 KC-inDomus ***@***.***> ha scritto:

**Reply by KC-inDomus** [reply]
leggiti questo, ti si apre un mondo
https://indomus.it/formazione/come-usare-gli-attuatori-broadlink-su-home-assistant/
